packages/authentication/src/keys.ts

```diff
@@ -5,7 +5,7 @@
 
 import {Strategy} from 'passport';
 import {AuthenticateFn, UserProfile} from './types';
-import {AuthenticationMetadata} from './decorators/authenticate.decorator';
+import {AuthenticationMetadata} from './decorators';
```

Comment:
Please note importing from the target file is not bad. Sometimes it's even necessary to avoid circular dependencies. If a module does not have side-effect code at top level, either way is fine.

Reply:
Absolutely, I'll keep your comment in my mind :)
Here this is just syntactic sugar to ease import of a "future" new decorator.